src/nl/help/faq/README.md

```diff
@@ -15,81 +15,83 @@ Er zijn geen fatsoenlijke manga-apps op **iOS** die _niet-opdringerige_ adverten
 Er zijn geen plannen om een formeel bedrijfsmodel vast te stellen. Zolang Patreon de kosten van een Apple Developer Account dekt, is de winst secundair. Het wordt op prijs gesteld om de app financieel te ondersteunen via Patreon, maar Paperback kost niets.
 
 ### Wanneer ben je van plan de app uit te brengen?
-Er is een publieke build van de applicatie op [GitHub](https://github.com/FaizanDurrani/Paperback-Public/releases). Het aanmelden voor een Apple Developer-account was moeilijk, maar hopelijk kunnen de TestFlight-builds binnenkort worden gestart. Na de TestFlight-builds hopen we Paperback in de App Store uit te brengen. Voor een gids om de applicatie te installeren, zie: 
+Er is een publieke build van de app op [GitHub](https://github.com/Paperback-iOS/app/releases). Het aanmelden voor een Apple Developer-account was moeilijk, maar hopelijk kunnen de TestFlight-builds binnenkort beginnen. Na de TestFlight-builds hopen we Paperback in de App Store uit te kunnen brengen. Voor een handleiding om de app te installeren:
 
-* [Paperback Installatiehandleiding](/nl/help/guides/getting-started)
+ * [Paperback Installatiehandleiding](/nl/help/guides/getting-started)
 
 ## Probleemoplossing
-Als je problemen hebt met de applicatie, kijk dan op deze pagina voordat je een bericht plaatst. Als je een probleem hebt dat hier niet wordt vermeld, gebruik dan het #support-kanaal op Discord om vragen te stellen (In het Engels).
+Als je problemen hebt met de app, kijk dan op deze pagina voordat je een bericht plaatst. Als je een probleem hebt dat hier niet wordt vermeld, gebruik dan het #support-kanaal op Discord om je vragen te stellen (In het Engels).
 
---- 
+---
 
 ### App loopt vast bij het opstarten
-**Publieke build** gebruikers moeten controleren of ze iOS 13.4+ gebruiken. **Beta / Patron build** gebruikers moeten Paper op Discord pingen in het #supporter-kanaal (In het Engels).
+Als je de **Publieke build** gebruikt, controleer dan of je iOS 13.4+ gebruikt. Als je de **Beta / Patron build** gebruikt, stuur een bericht op het Discord #supporter-kanaal en ping Paper (In het Engels).
 
 ---
 
 ### App loopt vast bij het volgen van manga op de iPad
-Dit is een bekende Apple-bug die is opgelost in iPadOS 13.4, update naar iOS 13.4 of vermijd het volgen van manga in Paperback en gebruik de MangaDex-website om manga te volgen (en vervolgens te synchroniseren).
+Dit is een bekende Apple-bug die is opgelost in iOS 13.4, update naar iOS 13.4 of vermijd het volgen van manga in Paperback en gebruik de MangaDex-website om manga te volgen (en vervolgens te synchroniseren).
```

Comment:
Seems like an issue in the English version.
We are talking about crash on iPad so I would write:
```suggestion
Dit is een bekende Apple-bug die is opgelost in iPadOS 13.4, update naar iPadOS 13.4 of vermijd het volgen van manga in Paperback en gebruik de MangaDex-website om manga te volgen (en vervolgens te synchroniseren).
```